### PR TITLE
Added support for array `service.id`

### PR DESCRIPTION
**Pull request by dekelev**
Added support for array `service.id` to support the composite PK feature in [feathers-objection](https://github.com/feathersjs-ecosystem/feathers-objection) & [feathers-cassandra](https://github.com/feathersjs-ecosystem/feathers-cassandra).

`service.docs.idType` option can now be also set to an array of types for the parameters in `service.id`.

`service.options.idSeparator` (default: `,`) is being used to separate the ID fields in the path, e.g. `/users/{firstId},{secondId}`.

See issue https://github.com/feathersjs-ecosystem/feathers-swagger/issues/194 for more details.

### CODE REVIEW

File: lib/v2/generator.js [hunk 1]
```diff
@@ -1,6 +1,24 @@
 const AbstractApiGenerator = require('../openapi');
 const utils = require('../utils');
 
+function idPathParameters (idName, idType, description) {
+  const idNames = Array.isArray(idName) ? idName : [idName];
+  const idTypes = Array.isArray(idType) ? idType : [idType];
+  const params = [];
+
+  for (const [key, name] of Object.entries(idNames)) {
```
Comment (Mairu) [inline lib/v2/generator.js:9]: Why you changed this from array to object iteration?

Comment (dekelev) [reply lib/v2/generator.js:9]: It's array iteration with key & value. it works with NodeJS 8 and up, so the iteration in the v1 branch is using a normal `for` loop. if you prefer you can apply that instead in the master branch.

Comment (Mairu) [reply lib/v2/generator.js:9]: Even though arrays are also objects, you should never call a function for objects on it, that transforms it into an array :)

Ok so I will merge the prepared branch.
Thank you very much for the PR.

**Comment by Mairu** [comment]
This was merged already.

**Pull request closed without merging**